zabbixsender/zabbixsvc/zabbixsvc.go

```diff
@@ -66,10 +93,20 @@ func (h *JSONHandler) HandlePost(w http.ResponseWriter, r *http.Request) {
 		value = "1"
 	}
 
+	var ammountOfResolved float64
+	if req.Status == "resolved" {
+		ammountOfResolved++
+	}
+
+	host := h.DefaultHost
+	for hostReceived, hostDefault := range h.DefaultHosts {
```

Comment:
Seriously wtf?  This is a `map`, you can just `host, ok := h.DefaultHosts[hostReceived]`, `if !ok... 

https://blog.golang.org/go-maps-in-action`